projects/core/src/cms/facade/cms.service.ts

```diff
@@ -12,11 +12,27 @@ import { CmsComponent } from '../../occ/occ-models/cms-component.models';
   providedIn: 'root'
 })
 export class CmsService {
+  private _launchInSmartEdit = false;
```

Comment:
The state about the presence of a CMSTicketId sort of duplicated in the SmarteditService and the CmsService.

Is there a way we can implement this without having the same state in 2 different services?

I presume it is done this way so the CmsService does not depend on SmartEditService.

Could the ticket be put in the ngrx store by the SmartEditService and the CmsService checks for the presence of a token?

Reply:
underscore indicates it is a private variable.

SmartEditModule is optional. Without it, we still should be able to get "whether app launch from smart edit". So, that's why I added the boolean flag in the CmsService.

I didn't put this ticketId in store, because it is not changed after received. If put in store, we have to select an observable.